.github/workflows/composite/maximize-build-space/action.yml

```diff
@@ -0,0 +1,18 @@
+---
+name: maximize-build-space
+description: Maximize build space on github runner
+
+runs:
+  using: composite
+  steps:
+    - name: Maximize build space
+      shell: bash
+      run: |
+        echo "Available storage before:"
+        df -h
+        echo "Removing /usr/share/dotnet, /usr/local/lib/android, /opt/ghc"
+        sudo rm -rf /usr/share/dotnet
+        sudo rm -rf /usr/local/lib/android
+        sudo rm -rf /opt/ghc
```

Comment:
- Do we have no password sudo permissions on the runner?
- Should we delete all folders in one call using line breaks?

Reply:
Sorry I missed a dash in the question. It should have been:
1. Do we have no-password sudo permissions on the runner?
I.e. if we can use `sudo` for the active user without password confirmation.

Reply:
Yes, it works as it is.